src/components/Filters/Filters.tsx

```diff
@@ -0,0 +1,109 @@
+import {
+  Box,
+  FormControl,
+  InputLabel,
+  MenuItem,
+  Select,
+  TextField,
+  ToggleButton,
+  ToggleButtonGroup,
+  Typography,
+} from "@mui/material";
+import { useTranslation } from "react-i18next";
+
+import { FILTERS_OPTIONS } from "../../constants/FiltersOptions";
+import { SORT_OPTIONS } from "../../constants/SortOptions";
+import { translatePath } from "../../constants/i18nPath";
+import { useFilters } from "../../hooks/useFilters";
+import { FILTERS } from "../../types/filterEnum";
+import SearchLink from "../SearchLink";
+import styles from "./Filters.module.scss";
+
+export const Filters = () => {
+  const { t } = useTranslation();
+
+  const { filter, query, sortState, handleChangeQuery, handleSortChange } =
+    useFilters();
+
+  return (
+    <Box className={styles.filterWrapper}>
+      <Box className={styles.filtersFormats}>
+        <Typography color="gray" fontSize={20} className={styles.filterTitle}>
+          {`${t(`${translatePath.filters}.movie_format`)}:`}
+        </Typography>
+        <ToggleButtonGroup
+          value={filter}
+          exclusive
+          className={styles.filterBtnGroup}
+        >
+          {FILTERS_OPTIONS.map((option) => (
+            <SearchLink
+              key={option}
+              params={{
+                filter: option === FILTERS.ALL ? null : option,
+                page: "1",
+              }}
+            >
+              <ToggleButton
+                className={`${option === filter ? styles.filterActiveBtn : ""}`}
```

Comment:
you can use option === filter && styles.filterActiveBtn

Reply:
done